_posts/2016/2016-12-15-hr-ui.md

```diff
@@ -0,0 +1,100 @@
+The project's name is Unity and we are developing a new shop software using frontend and backend technology.
+Currently, a beta version is on the market.
+We hope the new shop design, which includes simpler functionalities as well as a responsive storefront will be accepted and approved by the users.
+
+## How did you get to know ePages?
+
+Originally, I studied graphic design.
+I, then, decided that I wanted to head more into an e-commerce direction and work with programs such as HTML, LESS, JavaScript, and alike.
+At an open door event at Jena University, I got to know two employees from ePages.
+As we got engaged into a discussion, they told me about an open day at ePages' office in Jena about three weeks later.
+On that day, I got a very good impression of ePages as a company and employer, my potential colleagues and I was able to clarify some remaining questions.
+I applied for an internship, got invited for an interview and finally got accepted.
+
+## How was your onboarding and first time at ePages?
+
+I remember on my very first day, I got shown around and introduced to all teams, such as Application Management, QA, etc.
+Afterwards, I set up my computer and checked out our company's intranet [Confluence](https://de.atlassian.com/software/confluence) and tried to get familiar and remember all of my colleagues' names.
+
+In the first month, I attended a partner training in Hamburg and thereby got to know our office in Hamburg as well as my Hanseatic colleagues - a little sneak peak of what was about to become my future workplace and home so to speak.
+After the first week, I began to develop a style guide, which was a perfect way to get an overview of what kind of software we use and also to get in touch with my new colleagues.
+Thereafter, I firstly started with some smaller [Jira](https://de.atlassian.com/software/jira) tickets and fixed some bugs.
+
+## What are typical tasks of a UI-Designer and what technologies do you use to solve these?
+
+This varies quite a bit depending on the type of task.
+At the moment, for example, we are redesigning our existing software.
+Therefore, we design UI elements, or design and improve new features.
+Sometimes, we also refactor code - so as you see, my work greatly varies from conceptional and design-related work to code-related topics.
+
+I mostly work with LESS, HTML, CSS, GitHub, Source Tree and Photoshop for creating themes, e.g. the home page or category pages.
+At the moment, I don't work a lot with JavaScript, yet I always need to know some basics of ReactJS, whenever I need to change a bit of code in our templates.
+When it comes to our responsive themes, we don't work with Bootstraps, but developed our shop-software themes' mobile responsiveness ourselves.
+We do use Bootstraps though for our MBO pages, for example for the new product detail page, the general settings page and alike.
```

Comment:
We do use [Bootstrap](http://getbootstrap.com/) though for pages of our administration area, for example, for the new ..., and alike.